pkg/binary/envoytest/util.go

```diff
@@ -15,79 +15,144 @@
 package envoytest
 
 import (
-	"context"
 	"fmt"
+	"math/rand"
+	"os"
 	"path/filepath"
+	"sync"
 	"syscall"
+	"testing"
 	"time"
 
-	"github.com/mholt/archiver"
+	"github.com/mholt/archiver/v3"
+	"github.com/stretchr/testify/require"
 	"github.com/tetratelabs/log"
 
 	"github.com/tetratelabs/getenvoy/pkg/binary"
 	"github.com/tetratelabs/getenvoy/pkg/binary/envoy"
 	"github.com/tetratelabs/getenvoy/pkg/manifest"
 )
 
-// Reference indicates the default Envoy version to be used for testing
-var Reference = "standard:1.11.0"
+// Reference indicates the default Envoy version to be used for testing.
+// 1.12.7 is the last version to match the Istio version we are using (see go.mod)
+var Reference = "standard:1.12.7"
```

Comment:
no need to export? 😄

Reply:
will check and if I can unexport it I will after rebasing. good idea